app/routes/editor.js

```diff
@@ -79,6 +80,14 @@ export default AuthenticatedRoute.extend(ShortcutsRoute, {
             }
 
             this.get('controller').willTransition(transition);
+        },
+
+        saveTitle() {
+            this.get('controller.saveTitle')
+                .perform()
+                .then(function () {
+                    this.router._router.updateTitle();
```

Comment:
We shouldn't be accessing the private router instance like this. It would be better to use the existing system we have in place for working with titles, such as changing `titleToken` or `title` properties to be a function that extracts the post's title, then it would be possible to trigger the `collectTitleTokens` action rather than accessing private methods.